lib/wrappers/contributionReward.ts

```diff
@@ -154,7 +155,7 @@ export class ContributionRewardWrapper extends ProposalGeneratorBase {
       eventContext,
       this.contract.proposeContributionReward,
       [options.avatar,
-      Utils.SHA3(options.description),
+      options.descriptionIsHashed ? options.description : Utils.SHA3(options.description),
```

Comment:
makes a bit more sense to me to pass in whether you Arc.js to hash or not, so by default it would not be hashed, but this is ok too

Reply:
@tibetsprague I did it this way to avoid creating a breaking change.  For what it is worth, it has the most convenient option as the default.